src/v3/src/components/Widget/index.tsx

```diff
@@ -235,22 +234,8 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     if (!idxTransaction) {
       return;
     }
-
-    const { messages: newMessages = [] } = idxTransaction;
-
     events?.afterRender?.(getEventContext(idxTransaction));
-
-    // for multiple error messages
-    newMessages?.forEach((newMessage) => {
-      const { class: type, message: msg } = newMessage;
-      if (type === MessageType.ERROR) {
-        // error event
-        events?.afterError?.({
-          stepName: idxTransaction?.nextStep?.name,
-        }, { message: msg });
-      }
-    });
-
```

Comment:
Compared to v2, these errors should only be triggered after user interaction (model events), so I moved this into the useOnSubmit hook.